lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1638,6 +1638,10 @@ static bool ParseIRGenArgs(IRGenOptions &Opts, ArgList &Args,
   for (const auto &Lib : Args.getAllArgValues(options::OPT_autolink_library))
     Opts.LinkLibraries.push_back(LinkLibrary(Lib, LibraryKind::Library));
 
+  for (const auto &Lib : Args.getAllArgValues(options::OPT_autolink_library)) {
```

Comment:
```suggestion
  for (const auto &Lib : Args.getAllArgValues(options::OPT_public_autolink_library)) {
```

? Otherwise this new option doesn't appear to be read

Reply:
Thanks! My test case was also wrong 😅